### PR TITLE
Issue #118

### PR DESCRIPTION
**Pull request by kruchas91**
Fat arrow warning

### CODE REVIEW

**Comment by Yoric** [comment]
Can't automatically merge.
Can you rebase from upstream?

**Comment by kruchas91** [comment]
Is it good now ?

File: index.html [hunk 1]
```diff
@@ -16,9 +16,25 @@
 
 <link rel="stylesheet" type="text/css" href="css/style.css" />
 <link rel="stylesheet" type="text/css" href="css/menu.css" />
-<link rel="stylesheet" type="text/css" href="lib/Font-Awesome/css/font-awesome.min.css" />
```
Comment (Yoric) [inline index.html:19]: Please don't remove Font-Awesome.
(I know, I know, it may be my fault. Still, please don't remove it.)

File: index.html [hunk 2]
```diff
@@ -16,9 +16,25 @@
...
+
+<link rel="icon" type="image/png" href="img/icon60.png" />
```
Comment (Yoric) [inline index.html:20]: Please don't add the icon in this patch.
(same remark)

File: index.html [hunk 3]
```diff
@@ -16,9 +16,25 @@
...
+
+<script>
+
+function testFatArrowSupport() {
+	try {
+		eval("x => 0");
+	} catch (e) {
+		if (e instanceof SyntaxError) {
```
Comment (Yoric) [inline index.html:28]:
``` js
try {
  ...
} catch (e if e instanceof SyntaxError) {
  ...
}
```

File: index.html [hunk 4]
```diff
@@ -16,9 +16,25 @@
...
 <link rel="stylesheet" type="text/css" href="css/style.css" />
 <link rel="stylesheet" type="text/css" href="css/menu.css" />
-<link rel="stylesheet" type="text/css" href="lib/Font-Awesome/css/font-awesome.min.css" />
+
+<link rel="icon" type="image/png" href="img/icon60.png" />
+
+<script>
+
+function testFatArrowSupport() {
+	try {
```
Comment (Yoric) [inline index.html:25]: Please match the indentation of the rest of the source code.

File: index.html [hunk 5]
```diff
@@ -16,9 +16,25 @@
 
 <link rel="stylesheet" type="text/css" href="css/style.css" />
 <link rel="stylesheet" type="text/css" href="css/menu.css" />
-<link rel="stylesheet" type="text/css" href="lib/Font-Awesome/css/font-awesome.min.css" />
+
+<link rel="icon" type="image/png" href="img/icon60.png" />
+
+<script>
```
Comment (Yoric) [inline index.html:22]: Instead of putting your code in `index.html`, please put it in `app.js`.

File: index.html [hunk 6]
```diff
@@ -16,9 +16,25 @@
...
+
+function testFatArrowSupport() {
+	try {
+		eval("x => 0");
+	} catch (e) {
+		if (e instanceof SyntaxError) {
+			alert("Fat arrow are not supported.");
```
Comment (Yoric) [inline index.html:29]: That's an error message that users are never going to understand.
Something like `"This application requires a browser that implements ES6. Your browser seems to be missing some features of ES6."`

File: index.html [hunk 7]
```diff
@@ -16,9 +16,25 @@
...
+			alert("Fat arrow are not supported.");	
+		}
+	}
+}
+
+</script>
+
 </head>
-<body>
+<body onload="testFatArrowSupport()">
```
Comment (Yoric) [inline index.html:37]: Don't use `onload`. If you need to synchronize with loading, use `document.addEventListener("load", ...)`.

**Comment by Yoric** [comment]
I'll need a few changes before I can accept the pull request.

**Pull request closed without merging**